src/index.html

```diff
@@ -5,9 +5,48 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <section data-qa="card" class="card">
+      <img src="images/imac.jpeg"
+        alt="APPLE A1419 iMac 27 Retina
+        5K Monoblock (MNED2UA/A)"
+        class="card__image"
+      >
+
+      <h4 class="card__header">APPLE A1419 iMac 27" Retina
+        <br>
+        5K Monoblock (MNED2UA/A)
+      </h4>
+
+      <span class="card__code-info">Product code: 195434</span>
+
+      <div class="card__reviews-container">
+        <ul class="stars stars--4">
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+        </ul>
+
+        <span class="card__review-number">Reviews: 5</span>
+      </div>
+
+      <div class="card__price-container">
+        <span class="card__price">Price:</span>
+        <span class="card__price-amount">$2,199</span>
+      </div>
+
+      <a href="#"
+        data-qa="hover"
+        class="card__button"
+      >Buy
+      </a>
```

Comment:
Checklist:
14. [CODE STYLE] - If the HTML-element has long attribute values or number of attributes is more than 2 - start each one, including the first, on the new line with 2-space indentation related to tag. Tag’s closing bracket should be on the same level as opening one.
![image](https://github.com/mate-academy/layout_product-cards/assets/63243136/07c96441-86b9-4134-a70e-615ad5fc002f)